app/assets/stylesheets/forever_style_guide/modules/_button.scss

```diff
@@ -102,6 +102,10 @@
         border: 2px solid color('white');
       }
     }
+    &.btn-default.disabled {
+      border: none;
```

Comment:
I think you need to persist the border-width else there could be some weird alignment with other buttons.

Reply:
Corrected.